packages/suite/src/components/suite/Banners/index.tsx

```diff
@@ -13,8 +13,8 @@ import SafetyChecksBanner from './SafetyChecks';
 
 import type { Message } from '@suite-types/messageSystem';
 
-const Wrapper = styled.div`
-    z-index: 3;
+const Wrapper = styled.div<{ onTop?: boolean }>`
+    z-index: ${props => (props.onTop ? '10001' : '3')};
```

Comment:
It would be really nice to have those z-indexes defined somewhere in one config file and use it from there. Because if somebody in the future will need to increase z-index of the Overlay, he has no chance to know, that this has to be changed also. 

What about to create a followup issue for that?

Reply:
I definitely agree. There is a big mess with z-indexes in the code.